docs/src/overview/dedication.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Dedication Production Clusters"
+weight: 1
+---
+
+The Dedication Production Clusters is astonishing if you want a mature and extensive application and scalable solution in the cloud.
+It provides a set of Virtual Machine redundant configuration,  a minimum of three instances,  provisioned by Platform.sh for each customer.  Every service is replicated across all three virtual machines in a failover configuration (as opposed to sharding), allowing a site to remain up even if one of the VMs is lost entirely.
+Dedicated Production Clusters work nearly identically to Grid environments, and it does not require extra configuration on the client-side.  However, the services PostgreSQL or Kafka are not available on this plan.
+
+
+## Features
+
+* Domains available to dev environments
+* The master branch with three-Virtual Machine
+* Support to Data sync
+* Backup on Production does not freeze in the process.
+* You can deploy your application in any of our [supported cloud providers](development/faq.md#which-geographic-zones-does-platformsh-cover). 
+
+
+You cannot perform the following:
```

Comment:
Seems repeated. I would choose to include this in the intro  or here, and it makes a little more sense to me as a list here for some reason.

Reply:
I'll keep the information here.